tika-parsers/src/main/resources/org/apache/tika/parser/ocr/TesseractOCRConfig.properties

```diff
@@ -13,9 +13,20 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+# Tesseract properties
 tesseractPath=
 language=eng
 pageSegMode=1
 maxFileSizeToOcr=2147483647
 minFileSizeToOcr=0
-timeout=120
\ No newline at end of file
+timeout=120
+
+# properties for image processing
+# to enable processing, set enableProcessing to 1
+enableProcessing=0
```

Comment:
can we think of something besides "processing?" that's too generic. how about `enableImageProcessing`